_commands/basics/seq.md

```diff
@@ -0,0 +1,75 @@
+---
+---
+
+seq
+-------
+
+`seq`prints a sequence of numbers to standard ouput.
+
+~~~ bash
```

Comment:
Could you insert the output of the simple, correct operation case here, i.e.:

```
$ seq 3
1
2
3
```

Right before the detail line you have here?